stacks/data-lakehouse-iceberg-trino-spark/nifi.yaml

```diff
@@ -52,7 +52,7 @@ kind: Secret
 metadata:
   name: nifi-admin-credentials-secret
 stringData:
-  admin: {{ nifiAdminPassword }}
+  admin: {{nifiAdminPassword}}
```

Comment:
nit: We could revert that change because we ignored the `braces` rule for these files.

Reply:
```suggestion
  admin: {{ nifiAdminPassword }}
```